camera_intrinsic_matrix: [602.0226440429688,   0.0,             330.3763122558594,
                            0.0,             602.1392211914062, 235.12127685546875,
                            0.0,               0.0,               1.0]